chsdi/models/clientdata_dynamodb.py

```diff
@@ -73,7 +73,6 @@ def get(self):
 
 
 dynamodb_connection = DynamodbConnection()
-s3_connection = S3Connection()
```

Comment:
A connection is initialized only when we do:
```
s3_connection.get()
```

We use to share only the constructor to get those connections. Now that we have several possible connections on s3 we create a new object each time.